libs/design-system/src/lib/Components/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import { ReactComponent as ChevronRight } from '../../../../images/arrows/chevron-right.svg';
+import { ReactComponent as ChevronDown } from '../../../../images/arrows/chevron-down.svg';
+
+export interface IFAQ {
+  question: string;
+  answer: string;
+}
+
+export interface FAQProps {
+  faqList: IFAQ[];
+  variation: 'large' | 'small';
+}
+
+export function FAQ({ faqList, variation }: FAQProps) {
+  const [expanded, setExpanded] = useState(
+    new Array(faqList.length).fill(false)
+  );
+
+  const toggleExpansion = (index: number) => {
+    const newExpanded = [...expanded];
+    newExpanded[index] = !newExpanded[index];
+    setExpanded(newExpanded);
+  };
+
+  return (
+    <div className="bg-transparent">
```

Comment:
```suggestion
    <div>
```
```suggestion
    <div className="bg-transparent">
```

Reply:
I think this comment may have duplicated a few times on the PR review, but just so I'm clear, the review is just to remove the `classname="bg-transparent"` from the div?